modules/auth/saas/settings/subscriptions.tsx

```diff
@@ -6,12 +6,103 @@ import Spinner from "@/resources/containers/spinner";
 import Service from "@/core/module/service.factory";
 import { useRouter } from "next/router";
 import moment from "moment";
+import Table from "@/resources/table";
 
 import { FaMoneyBillTransfer } from "react-icons/fa6";
 
+const data: TableDataModel[] = [
+  {
+    id: "Invoice 1",
+    date: new Date().toLocaleDateString(),
+    plan: "Basic",
+    amount: "€ 200",
+  },
+  {
+    id: "Invoice 2",
+    date: new Date().toLocaleDateString(),
+    plan: "Basic",
+    amount: "€ 150",
+  },
+];
+
+[3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20].forEach(
+  (i) => {
+    data.push({
+      id: `Invoice ${i}`,
+      date: new Date().toLocaleDateString(),
+      plan: "Basic",
+      amount: "€ 150",
+    });
+  }
+);
+
+const PaymentHistory: React.FC = () => {
+  const columns: ColumnConfig<TableDataModel>[] = [
+    { title: "Invoice ID", dataIndex: "id", sorter: true },
+    {
+      title: "Date",
```

Comment:
Remember to make the titles translated, the lang now is in the same subscription file 'subscription-settings'

Reply:
Done, that was on my list